.github/CONTRIBUTING.md

```diff
@@ -30,6 +30,36 @@ Our kanban board is public and available [here](https://github.com/orgs/vazco/pr
 - Make sure you have added the necessary tests for your changes. Do not worry though, the Codecov bot will report it in the pull request.
 - Make sure your code passes _all_ tests: `npm test`.
 
+### Development
+
+For the best developer experience (DX) when developing locally it is recommended to start `build` and `test` in a `--watch` mode.
+
+In the root directory...
```

Comment:
This sounds a little strange. Maybe reword this to a direct command like "Run the commands below in the root directory."